<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2026</width>
    <height>1202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2022 CvDl Hw2_05</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupResnet50">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>341</width>
      <height>1111</height>
     </rect>
    </property>
    <property name="title">
     <string>5. Resnet50</string>
    </property>
    <widget class="QPushButton" name="pushButtonShowImages">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Show Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowModelStructure">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>3. Show Model Structure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowDistribution">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2. Show Distribution</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowComparsion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>770</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>4. Show Comparsion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowInference">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>950</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>5. Inference</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadImage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>1301</width>
      <height>981</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../cvdl_hw1_04/Dataset_CvDl_Hw1_2/Q4_images/traffics.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textArea">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>920</y>
      <width>1301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
